share/governance.md

```diff
@@ -105,10 +105,9 @@ election. There are 5 members of the PMC. Terms are 1 year.
 
 The PMC doesn't have specific roles or a chairperson.
 
-At the moment, IRC is our main communication channel. PMC members should be
-present in our `#voxpupuli-pmc` channel on [libera.chat](https://libera.chat).
-Also we have some gpg-encrypted credentials and a few repositories require
-gpg-signed commits, so a PMC member also needs a gpg key.
+At the moment, IRC is our main communication channel.
+PMC members should be present in our `#voxpupuli-pmc` channel on [libera.chat](https://libera.chat).
+Also we have some gpg-encrypted credentials and a few repositories require gpg- or ssh-signed commits, so a PMC member also needs a gpg key.
```

Comment:
```suggestion
Also we have some gpg-encrypted credentials and a few repositories require gpg- or ssh-signed commits, so a PMC member also needs keys.
```

They need both keys, right?

Reply:
Technically, they need a gpg key to encrypt the puppet_credentials file. They could authenticate with github via HTTPS= token and sign commits with GPG, so an ssh key isn't required. But your life is much easier with an ssh key :D